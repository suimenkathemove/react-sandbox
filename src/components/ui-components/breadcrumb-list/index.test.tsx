import { cleanup, render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import 'jest-styled-components';
import { BreadcrumbList } from './';

const setup = () => render(<BreadcrumbList {...defaultProps} />);

describe('BreadcrumbList', () => {
  afterEach(() => {
    cleanup();
  });

  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
