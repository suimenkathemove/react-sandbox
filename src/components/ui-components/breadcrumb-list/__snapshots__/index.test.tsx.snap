// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadcrumbList render 1`] = `
<DocumentFragment>
  .c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<nav>
    <ol
      class="c0"
    >
      <li
        class="c1"
      >
        <span>
          1
        </span>
        &gt;
      </li>
      <li
        class="c1"
      >
        <span>
          2
        </span>
        &gt;
      </li>
      <li
        class="c1"
      >
        <span>
          3
        </span>
      </li>
    </ol>
  </nav>
</DocumentFragment>
`;
